### ISSUE TITLE
CBD - ws json Config to MZ@v6-jc

### ISSUE THREAD

**Issue by BRaimbault** [psi-mis/connectWebapp#43]
update to new version

**Comment by jturtler** [comment]
I have check this version - to see if I can use it.

It seems the Household Delivery (CwS - MZ - *Household delivery) values are changed to be either, 'PIL', 'INJ', ''.  

        {
          "id": "wxAMVtMTUtp",
          "defaultName": "CwS - MZ - *Household delivery",
          "outDataName": "CBD_HouseHold_Delivery",          
          "evaluate": "[acdv.currentMethod] === 'PIL' ? [acdv.householdDelivery_PIL] : [acdv.currentMethod] === 'INJ' ? [acdv.householdDelivery_INJ] : ''"
        },

I currently do expect below values: 
 - "YES-Microgynon"
 - "YES-Microlut"
 - "YES-Injectables"
 - "NO"

The first 3 are proper HouseHold Delivery values.

So, what values should I expect now for HouseHold Delivery values?  Will there be 'NO'?  Or would it be replaced as empty value ('')?

**Comment by BRaimbault** [comment]
You should expect:
- "YES-Microgynon"
- "YES-Microlut"
- "YES-Injectables"
- "NO"
If those are not the values then the formula has a problem.
I add some background info on how the formula is build below.

In android, to control the list of available options for a given question, we duplicated it:
- [acdv.householdDelivery_PIL] can take the values "YES-Microgynon" / "YES-Microlut" / "NO"
- [acdv.householdDelivery_INJ] can take the values "YES-SayanaPress" / "NO"

The rule to display one or the other in android is:
- Show [acdv.householdDelivery_PIL] if [acdv.currentMethod] === 'PIL'
- Show [acdv.householdDelivery_INJ] if [acdv.currentMethod] === 'INJ'

With the short hand syntax: 
````
 "If Condition" ? "Val TRUE" : "Else FALSE"
````
You can simply create Else If the following way:
````
 "If Condition 1" ? "Val 1 TRUE" : "Else If Condition 2" ? "Val 2 TRUE" : "Else FALSE"
````

So basically the formula should:
- Take [acdv.householdDelivery_PIL] value if [acdv.currentMethod] === 'PIL'
- Take [acdv.householdDelivery_INJ] value if [acdv.currentMethod] === 'INJ'

Let me know if you spot any issue.
